scripts/config.ts

```diff
@@ -48,7 +48,7 @@ const NETWORKS = {
 		},
 		TREASURY_FEE_PERCENT: 100, // in BPS i.e 0.1%
 		GATEWAY_CONTRACT: "0x1FA0EE7F9410F6fa49B7AD5Da72Cf01647090028",
-		IMPERSONATE_ACCOUNT: "",
+		IMPLEMENTATION: "",
```

Comment:
why is this change made for only binance smart chain and base sepolia?

why not on other networks?

Reply:
The reason is because i used it to test the script, if you check, you will realised that it actually auto implemented from the script during deployment, so anytime an implementation contract is deployed it update `GATEWAY_IMPLEMENTATION` in the config with the new address.